nasa-gibs/worldview
# wv-2583.Gtranslate_error

## Description
When WorldView is translated using the google translate extension interaction with the sidebar can lead to a crash.
Fixes #WV-2583

## How To Test
To Reproduce the issue (in production):

- Install Google Translate extension (https://chrome.google.com/webstore/detail/google-translate/aapbdbdomjkkjkaonfhkkikfgjllcleb?hl=en)
- Load Worldview using Chrome 
- Click on the extension and "translate this page" to non-english
- Change the left sidebar from "Layers" to "Events"
- You will see an error in production. To re-trigger the error, you need to open a fresh tab & restart the process.

Follow the same steps above with this PR & confirm that the crash no longer occurs.

@nasa-gibs/worldview
